TTS/auto_tts/model_hub.py

```diff
@@ -0,0 +1,543 @@
+import os
+
+from TTS.tts.configs.glow_tts_config import GlowTTSConfig
+from TTS.tts.configs.speedy_speech_config import SpeedySpeechArgs, SpeedySpeechConfig
+from TTS.tts.configs.tacotron2_config import Tacotron2Config
+from TTS.tts.configs.vits_config import VitsConfig
+from TTS.vocoder.configs.hifigan_config import HifiganConfig
+from TTS.vocoder.configs.multiband_melgan_config import MultibandMelganConfig
+from TTS.vocoder.configs.univnet_config import UnivnetConfig
+from TTS.vocoder.configs.wavegrad_config import WavegradConfig
+
+
+class TtsModels:
+    """
+    This is a class that holds all the model configs.
+    If you want to add a new recipe this is where you would add your model config
+    made this so the complete recipes file doesnt look so cluttered. If you would like
+    to contribute a model for a recipe look at the usage on how you should format the function.
+
+    usage:
+            def model_name:
+                config = model_config(...)
+                return config
+    """
+
+    def __init__(
+        self, batch_size, mixed_precision, learning_rate, epochs, output_path=os.path.dirname(os.path.abspath(__file__))
+    ):
+
+        self.batch_size = batch_size
+        self.output_path = output_path
+        self.mixed_precision = mixed_precision
+        self.learning_rate = learning_rate
+        self.epochs = epochs
+
+    def single_speaker_tacotron2_base(
+        self, audio, dataset, dla=0.25, pla=0.25, ga=5.0, forward_attn=True, location_attn=True
+    ):
+        config = Tacotron2Config(
```

Comment:
How about fetching these configs from the real `recipes`  when exist to reduce the decoupling